packages/workers-shared/router-worker/wrangler.toml

```diff
@@ -12,6 +12,7 @@ account_id = "0f1b8aa119a907021f659042f95ea9ba"
 workers_dev = false
 main = "src/index.ts"
 compatibility_date = "2024-07-31"
+compatibility_flags = ["nodejs_compat"]
```

Comment:
Can you add a comment explaining that this is only here because of Vitest?
It makes me said that we are turning this on for a Worker that is deeply embedded in the standard pipeline for many of our customer's projects going forward.

Reply:
It might make sense to also add `no_nodejs_compat_v2` here to avoid filling this worker with unnecessary global polyfills from `process`, `console`, etc.: https://github.com/cloudflare/workers-sdk/issues/7108

Reply:
Thanks both! Updated PR based on comments :)